series/series_test.go

```diff
@@ -1604,12 +1603,12 @@ func TestSeries_Map(t *testing.T) {
 		return Element(result)
 	}
 
-		for testnum, test := range tests {
+	for testnum, test := range tests {
 		switch test.series.Type() {
 		case Bool:
 			expected := test.expected
 			received := test.series.Map(and)
-			for i := 0 ; i<expected.Len() ; i++ {
+			for i := 0; i < expected.Len(); i++ {
 				e, _ := expected.Elem(i).Bool()
 				r, _ := received.Elem(i).Bool()
 
```

Comment:
I'm surprised to see so many changes here, I'm guessing because before there was no gofmt on this code? Unfortunately this many changes make it a bit difficult to review. Can you fix this to include only the changes relevant to your code?

We should fix the styling issues in all files on a separate PR.